frontend/src/pages/Projects.tsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect } from "react"
+
+
+export default function Projects() {
+
+    useEffect(()=>{
+        document.title = "OWASP Projects"
+        const fetchApiData = async () => {
+            try {
+                const response = await fetch(`http://localhost:8000/api/v1/owasp/search/project?q=`)
```

Comment:
It looks like it's time for frontend env variables introduction. We need an BACKEND_URL / API_URL value for the new pages. Let's focus on that in a separate PR.

Reply:
I will set those env for frontend in the next PR